taxi/views.py

```diff
@@ -1,3 +1,20 @@
+from django.http import HttpRequest, HttpResponse
+
 from django.shortcuts import render
 
+from taxi.models import Driver, Manufacturer, Car
```

Comment:
There should be a blank line between the import of `render` from `django.shortcuts` and the import of models from `taxi.models` to separate third-party imports from local application imports, as per the checklist.